src/utils/makePaths.js

```diff
@@ -1,6 +1,6 @@
 const path = require('path')
 
 module.exports.makePaths = cwd => {
-    const NODE_MODULES = path.join(cwd, 'node_modules')
-    return { NODE_MODULES }
+    const PACKAGE_JSON = path.join(cwd, 'package.json')
```

Comment:
It might be nice if we used `find-up` to run the script from the nearest `package.json`, but that might also be confusing when using file globs relative to the actual `cwd`?

Reply:
I like the idea, sounds more like a future PR though